### DISCUSSION TITLE
Screen in default mode

### DISCUSSION THREAD

**Discussion by Lagonow** [pmgl/microstudio#218] [General]
in file >> static/js/runtime/screen.js
It is creating a Screen object.
The width and height values are very large compared to other rendering options ( 200x200, 1000x800 ).
Additionally, they seem to have swapped places.
The typical value is 1920x1080 and there it is 1080x1920

     this.canvas.width = 1080;
     this.canvas.height = 1920;

**Comment by CoderGuyTedd** [comment]
It should be relative, right?
